src/zoo.ml

```diff
@@ -197,7 +196,8 @@ struct
         | None -> fatal_error "I am sorry but this language has no interactive toplevel."
       in
       Format.printf "%s -- programming languages zoo@\n" L.name ;
-      Format.printf "Type %s to exit@." eof ;
+      Format.printf "Type %s to exit@\n" eof ;
```

Comment:
`Format.printf "Split long lines with \\, type %s to exit.@\n" L.name`

I feel this stuff is just clutter. Can we maybe put it in the usage message?